### PR TITLE
Add test for the one-argument version of scroll() and fix message for…

### PR DESCRIPTION
**Pull request by fred-wang**
… the two-argument version.

See https://drafts.csswg.org/cssom-view/#dom-element-scroll

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5632)

### CODE REVIEW

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5632/)

File: cssom-view/elementScroll.html [hunk 1]
```diff
@@ -54,11 +54,33 @@
...
+
+            section.scroll({top: 75});
+
+            assert_equals(section.scrollLeft, 85, "scrollLeft should stay at 85");
+            assert_equals(section.scrollTop, 75, "changed scrollLeft should be 75");
+
+            section.scroll({});
+
+            assert_equals(section.scrollLeft, 85, "scrollLeft should stay at 85");
+            assert_equals(section.scrollTop, 75, "scrollTop should stay at 75");
```
Comment (zcorpan) [inline cssom-view/elementScroll.html]: Can you also test `section.scroll();` (should be equivalent to `section.scroll({});`)

**Comment by zcorpan** [comment]
Would be nice with dictionary argument tests for `scrollTo` and `scrollBy` as well. Otherwise LGTM

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/224284891)

# Lint #

Passed

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/224284893)

# Firefox (nightly channel) #
Testing web-platform-tests at revision a85bca4d15d1ef4328a73bf3119e2bd654f05f1f
Using browser at version BuildID 20170420100256; SourceStamp 27311156637f9b5d4504373967e01c4241902ae7
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5632/cssom-view/elementScroll.html">/cssom-view/elementScroll.html</a></summary>


|                   Subtest                   | Results |                                  Messages                                 |
|---------------------------------------------|---------|---------------------------------------------------------------------------|
|                                             | OK      |                                                                           |
| `Element scrollTop/Left getter/setter test` | PASS    |                                                                           |
| `Element scroll test`                       | PASS    |                                                                           |
| `Element scroll test (one argument)`        | PASS    |                                                                           |
| `Element scrollTo test`                     | PASS    |                                                                           |
| `cssom-view - elementScroll`                | PASS    |                                                                           |
| `Element scroll maximum test`               | FAIL    | `assert_equals: changed scrollTop should be 636 expected 636 but got 970` |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/224284899)

# Chrome (unstable channel) #
Testing web-platform-tests at revision a85bca4d15d1ef4328a73bf3119e2bd654f05f1f
Using browser at version 59.0.3071.15 dev
Starting 10 test iterations

**Comment by fred-wang** [comment]
@zcorpan Thanks for the feedback, I push one more commit. Can you please check again?

**Comment by zcorpan** [comment]
LGTM.

It seems `Element scroll maximum test` fails in Chrome and Firefox... filed https://github.com/w3c/web-platform-tests/issues/5634

**Review by zcorpan — APPROVED** [review]

**Comment by fred-wang** [comment]
@zcorpan Thanks. Yes, I noticed the failure too and wanted to investigate that. From a quick look, it was not obvious how the expected values are determined.

**Pull request merged**